manifests/supervisorcluster/1.29/cns-csi.yaml

```diff
@@ -495,6 +499,13 @@ spec:
           hostPath:
             path: /etc/vmware/wcp/tls/
             type: Directory
+        - name: client-ca
+          configMap:
+            name: kube-root-ca.crt
+            defaultMode: 420
+            items:
+            - key: "ca.crt"
+              path: "ca.crt"
```

Comment:
Could you provide a more detailed explanation about this? who creates this configmap?

Reply:
We will use the Kubernetes root CA for client authentication as client(APIserver) cert here is created with Kubernetes root CA used during cluster formation (see https://github-vcf.devops.broadcom.net/vcf/cayman_photon/pull/73)
The client CA will be available at /etc/kubernetes/pki/ca.crt on CPVMs and also available in a cm in all namespaces by the name kube-root-ca.crt. Hence we mount that configmap here so that the CA can be used while verifying the client CA certificate.
This will be common procedure followed by all supervisor webhooks.
Details can be found at https://vmw-confluence.broadcom.net/display/WCP/%5BBug+3448018%5D+Security+issues+in+Supervisor+Webhooks#id-[Bug3448018]SecurityissuesinSupervisorWebhooks-Conclusion